### PR TITLE
Release v0.10.0

### PR DESCRIPTION
**Pull request by sysadmind**
Cut a stable release before larger refactoring changes.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+## 0.10.0 / 2021-07-07
+* [ENHANCEMENT] Add ability to set included databases when autoDiscoverDatabases is enabled #499
```
Comment (SuperQ) [inline CHANGELOG.md:2]: Extra space needed here.

```suggestion

* [ENHANCEMENT] Add ability to set included databases when autoDiscoverDatabases is enabled #499
```

**Comment by SuperQ** [comment]
Can you add a `make update-go-deps` to this?

Although, it looks like lib/pq changes some string formatting:

```
$ make test
>> running all tests
GO111MODULE=on go test -race  ./...

----------------------------------------------------------------------
FAIL: postgres_exporter_test.go:218: FunctionalSuite.TestParseFingerprint

postgres_exporter_test.go:254:
    c.Assert(f, Equals, cs.fingerprint)
... obtained string = "'localhost':'55432'"
... expected string = "localhost:55432"

```

**Comment by sysadmind** [comment]
I bumped the modules, but lib/pq v1.10.2 seems to break illumos builds. I'll roll this back to v1.10.1 for now and submit an issue upstream.

File: go.mod [hunk 2]
```diff
@@ -1,15 +1,15 @@
 module github.com/prometheus-community/postgres_exporter
 
-go 1.14
+go 1.16
```
Comment (SuperQ) [inline go.mod:3]: We don't usually change this unless we need to for language feature reasons. Does this not build on 1.14 or 1.15 anymore?

Comment (sysadmind) [reply go.mod:3]: I think I changed this trying to figure out why illumos wouldn't build. I'll roll it back. Thanks for the catch.

**Review by SuperQ — APPROVED** [review]
LGTM

**Comment by sysadmind** [comment]
@SuperQ You will need to merge and tag. I don't have permissions on this repo

**Comment by SuperQ** [comment]
I've added the tag for the release, it should pop out of CI in a bit.

**Pull request merged**